apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.ts

```diff
@@ -123,35 +117,41 @@ export class EditApplicationFormItemDialogComponent implements OnInit {
 
   getDestinationAndSourceAttributes() {
     this.translateService.get('DIALOGS.APPLICATION_FORM_EDIT_ITEM.NO_SELECTED_ITEM').subscribe( noItem => {
-      this.sourceAttributes.push(new SelectionItem(noItem, ''));
-      this.destinationAttributes.push(new SelectionItem(noItem, ''));
+      const emptyItem: SelectionItem = new SelectionItem(noItem, '');
+      this.sourceAttributes.push(emptyItem);
+      this.destinationAttributes.push(emptyItem);
+      this.sourceAtt = emptyItem;
+      this.destinationAtt = emptyItem;
```

Comment:
with adding new search select component this whole logic could be I believe in that component and send only to that component `attributeDefinitions` and how the attributes are viewed for users that up to that component. We could simplify this whole edit-application-form-item-dialog component